setup.py

```diff
@@ -48,5 +50,5 @@
         'Topic :: Software Development :: Quality Assurance',
         'Topic :: Software Development :: Testing',
     ],
-    install_requires=['selenium ~= 4.1'],
+    install_requires=['selenium >= 4.1, < 4.10'],
```

Comment:
not sure why this version should be set here. I though Pipfile is the only source of truth

Reply:
Hm, Pipfile is only in this repository, I thought it was for dev purpose. The package does not include it

Reply:
Yes, you are right. Sorry for the confusion: https://gist.github.com/dpboard/887b74f242605c3a409b90f0cf706531

Although I am still not quite convinced about version locking strategy

Reply:
(sorry, my another comment was reflected slowly)